app/scripts/app.js

```diff
@@ -56,7 +56,7 @@ angular
         controller: 'SubmitCtrl'
       })
       .when('/detail/:EditType/:EditId', {
```

Comment:
I intentionally left them uppercase so that when we reference them in the controller via $routeParams, that it would be easier to tell where they came from since I reuse editType and editId later on in that controller.